app/src/main/java/org/mozilla/focus/cfr/CfrPopup.kt

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.cfr
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.PopupWindow
+import androidx.core.content.ContextCompat
+import org.mozilla.focus.databinding.PopupCfrBinding
+
+class CfrPopup(
+    private val parentView: ViewGroup,
+    private val shapeId: Int,
+    private val onDismissCallback: () -> Unit
+) : PopupWindow() {
+    private lateinit var binding: PopupCfrBinding
+
+    init {
+        initializeLayout()
+    }
+
+    private fun initializeLayout() {
+        val layoutInflater =
+            parentView.context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
+        binding = PopupCfrBinding.inflate(layoutInflater, parentView, false)
```

Comment:
```suggestion
binding = PopupCfrBinding.inflate(LayoutInflater.from(parentView.context), parentView, false)
```
Just for consistency. Seems like `LayoutInflater.from(..)` is preferred and used in many other places.